### PR TITLE
Removing deprecation warnings

### PR DESCRIPTION
**Pull request by tiomoreno**
Change from inspect.getargspec() to inspect.getfullargspec() to remove deprecation warnings

### CODE REVIEW

**Comment by quiqueporta** [comment]
Thanks @hugoleodev  ;)
Do you have any example to test it?

**Comment by tiomoreno** [comment]
@quiqueporta When you use getargspec on python 3, the following warning will appear 
 
DeprecationWarning: inspect.getargspec() is deprecated, use inspect.signature() or inspect.getfullargspec()

In my case I use your project with python 3

**Comment by quiqueporta** [comment]
Ok, I will create another release with this change
Thanks :wink:

File: simple_value_object/value_object.py [hunk 1]
```diff
@@ -120,22 +120,27 @@ class ArgsSpec(object):
 
     def __init__(self, method):
         try:
-            self._args, self._varargs, self._keywords, self._defaults = getargspec(method)
+            if sys.version_info.major == 2:
+                self.argspec = inspect.getargspec(method)
```
Comment (quiqueporta) [inline simple_value_object/value_object.py]: Can you define **argspec**, and **varkw** private?

**Review by quiqueporta — CHANGES_REQUESTED** [review]

**Review by quiqueporta — APPROVED** [review]

**Pull request merged**